beaver_store/lib/src/storage_service/gcloud_storage_service.dart

```diff
@@ -71,24 +73,54 @@ class BeaverBuild extends Model {
 class GCloudStorageService implements StorageService {
   DatastoreDB _db;
 
+  Future<String> _allocateProjectId() async {
```

Comment:
Just curious.

This return value will be project id. Correct?
What is the format of this string? Show me the example, please.